litmus/ARMCompile_litmus.ml

```diff
@@ -79,6 +83,20 @@ module Make(V:Constant.S)(C:Config) =
         inputs = [] ;
         outputs = [r1]; cond=is_cond c; }
 
+    let movw r1 i =
+      let memo = "movw" in
+      { empty_ins with
+        memo = sprintf "%s ^o0,#%i" memo i ;
+        inputs = [] ;
+        outputs = [r1]; }
+
+    let movt r1 i =
+      let memo = "movt" in
+      { empty_ins with
+        memo = sprintf "%s ^o0,#%i" memo i ;
+        inputs = [] ;
```

Comment:
The `MOVT` instruction reads its register argument, this is important for `litmus7` to output code that initialize this register:
```suggestion
        inputs = [r1] ;
```